other/curves.json

```diff
@@ -419,6 +419,38 @@
       "order": "0x3fffffffffffffffffffffffffffffffffffffffffffffffffffffff7cca23e9c44edb49aed63690216cc2728dc58f552378c292ab5844f3",
       "cofactor": "0x04"
     },
+    {
+      "name": "Curve41417",
+      "category": "other",
+      "desc": "Curve from https://cr.yp.to/talks/2013.09.16/slides-djb-20130916-a4.pdf",
+      "field": {
+        "type": "Prime",
+        "p": "0x3fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffef", //podle DB i safecurves
+        "bits": 414
+      },
+      "form": "TwistedEdwards",
+      "params": {
+        "a": {
+          "raw": "0x01"
+        },
+        "d": {
+          "raw": "0xe21"
+        }
+      },
+      "generator": {
+        "x": {
+          "raw": "0x1a334905141443300218c0631c326e5fcd46369f44c03ec7f57ff35498a4ab4d6d6ba111301a73faa8537c64c4fd3812f3cbc595" //db i safe
+        },
+        "y": {
+          "raw": "0x22"
+        }
+      },
+      "order": "0x7ffffffffffffffffffffffffffffffffffffffffffffffffffeb3cc92414cf706022b36f1c0338ad63cf181b0e71a5e106af79", //db
+      "cofactor": "0x08",
+	  "aliases": [
+        "other/Curve3617"
```

Comment:
This alias actually doesn't exist, the DB only supports to alias between existing curves. Maybe we will add support for adding purely name aliases that show up.